resources/dicts/patrols/general/border.json

```
@@ -2176,5 +2176,298 @@
             }
         ]
 
-    }
-]
\ No newline at end of file
+    },
+{
+    "patrol_id": "gen_bord_appmission",
+    "biome": ["any"],
+    "season": ["any"],
+    "types": ["border"],
+    "tags": ["other_clan"],
+    "patrol_art": "gen_train_talkingAW",
+    "min_cats": 2,
+    "max_cats": 2,
+    "min_max_status": {
+        "warrior": [1, 1],
+        "apprentice": [1, 1]
+    },
+    "weight": 20,
+    "chance_of_success": 20,
+    "intro_text": "p_l can't help but think that app1 has been distracted on this patrol, trailing behind and admiring any aimless insect that hovers nearby. {PRONOUN/p_l/subject/CAP} {VERB/p_l/decide/decides} that app1 could do with a <i>special</i> mission to get {PRONOUN/app1/object} back on track.",
+    "decline_text": "After a bit of thought, p_l decides that getting an apprentice to sneak across the border probably wasn't the best idea... however irritating {PRONOUN/app1/subject} {VERB/app1/were/was} being.",
```

```suggestion
    "decline_text": "After a bit of thought, p_l decides that getting an apprentice to sneak across the border probably isn't the best idea... however irritating {PRONOUN/app1/subject} {VERB/app1/were/was} being.",
```
small rewording to make it clearer that p_l *didn't* ask the app to cross the border.